src/panels/lovelace/entity-rows/hui-sensor-entity-row.ts

```diff
@@ -48,23 +52,34 @@ class HuiSensorEntityRow extends LitElement implements EntityRow {
     return html`
       ${this.renderStyle()}
       <hui-generic-entity-row .hass="${this.hass}" .config="${this._config}">
-        <div>
-          ${
-            stateObj.attributes.device_class === "timestamp"
-              ? html`
-                  <hui-timestamp-display
-                    .hass="${this.hass}"
-                    .ts="${new Date(stateObj.state)}"
-                    .format="${this._config.format}"
-                  ></hui-timestamp-display>
-                `
-              : stateObj.state
-          }
-        </div>
+        <div>${this.renderValue(stateObj)}</div>
       </hui-generic-entity-row>
     `;
   }
 
+  private renderValue(stateObj): TemplateResult {
+    const display = this._config!.display || "state";
+
+    let value;
+    if (display === "state") {
+      value = computeStateDisplay(this.localize, stateObj, this.hass!.language);
+    } else if (display.startsWith("attributes.")) {
```

Comment:
I would prefer `attributes:`

Reply:
I picked `attributes.` because it fits with how the entity object is referenced in both javascript and jinja templates, e.g. `state.light.my_light.state`(`display: state`), `state.light.my_light.last_changed` (`display: last_changed`), `state.light.my_light.attributes.brightness` (`display: attributes.brightness`).